src/drivers/spektrum_rc/spektrum_rc.cpp

```diff
@@ -50,8 +50,11 @@
 #include <uORB/topics/input_rc.h>
 
 // Snapdraogon: use J12 (next to J13, power module side)
+#if defined(__PX4_QURT)
+#define SPEKTRUM_UART_DEVICE_PATH "/dev/tty-1"
+#else
 #define SPEKTRUM_UART_DEVICE_PATH "/dev/tty-3"
```

Comment:
@dagar -  that may make good sense. I think I would want to understand the use cases better and the memory footprint if it is broken out.

The FMU is doing all the RC protocols. Do there other systems do the same (all protocols)? Would an RC app, with compile time conditional protocol be a good replacement for the FMU code the FMU that is doing this?